src/pages/chat/Chat.tsx

```diff
@@ -21,60 +21,74 @@ import ChatTextArea from '~/containers/chat/chat-text-area/ChatTextArea'
 
 import { defaultResponses } from '~/constants'
 import { styles } from '~/pages/chat/Chat.styles'
-import {
-  ChatResponse,
-  MessageInterface,
-  PositionEnum,
-  GetMessagesParams
-} from '~/types'
+import { ChatResponse, MessageInterface, PositionEnum } from '~/types'
 
 const Chat = () => {
   const { t } = useTranslation()
   const { isMobile } = useBreakpoints()
   const { openDrawer, closeDrawer, isOpen } = useDrawer()
   const [selectedChat, setSelectedChat] = useState<ChatResponse | null>(null)
-  const [textAreaValue, setTextAreaValueValue] = useState<string>('')
+  const [messages, setMessages] = useState<MessageInterface[] | []>([])
+  const [textAreaValue, setTextAreaValue] = useState<string>('')
   const scrollRef = useRef<HTMLDivElement | null>(null)
 
   const openChatsHandler = () => {
     openDrawer()
   }
 
   const onTextAreaChange = (e: ChangeEvent<HTMLInputElement>) => {
-    setTextAreaValueValue(e.target.value)
+    setTextAreaValue(e.target.value)
   }
 
+  const onMessagesResponse = useCallback(
+    (response: MessageInterface[]) => setMessages(response),
+    [setMessages]
+  )
+
   const getChats = useCallback(() => chatService.getChats(), [])
 
+  const sendMessage = useCallback(
+    () =>
+      messageService.sendMessage({
+        chatId: selectedChat?._id ?? '',
+        text: textAreaValue
+      }),
+    [selectedChat?._id, textAreaValue]
+  )
+
   const getMessages = useCallback(
-    (params?: GetMessagesParams) => messageService.getMessages(params),
-    []
+    () => messageService.getMessages({ chatId: selectedChat?._id ?? '' }),
+    [selectedChat?._id]
   )
 
   const { response: listOfChats, loading } = useAxios({
     service: getChats,
     defaultResponse: defaultResponses.array
   })
 
-  const {
-    response: messages,
-    loading: messagesLoad,
-    fetchData
-  } = useAxios({
+  const { fetchData } = useAxios({
     service: getMessages,
+    onResponse: onMessagesResponse,
     defaultResponse: defaultResponses.array,
     fetchOnMount: false
   })
 
+  const onMessageSend = async () => {
+    setTextAreaValue('')
+    await sendMessage()
+    void fetchData()
```

Comment:
you can don't use void if you have async function, just
```javascript
await fetchData()
```

Reply:
changed